app/controllers/episodes_controller.rb

```diff
@@ -1,11 +1,48 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
+  before_action :set_episode, only: %i[edit show update]
 
   def index
     @episodes = Episode.order(created_at: :desc).page params[:page]
   end
 
-  def show
+  def show; end
+
+  def edit; end
+
+  def create
+    @episode = Episode.new(episode_params)
```

Comment:
Try it in the next way
```ruby
@episode = Episode.create(created_by: current_user, **episode_params)

if @episode.valid?
...
```

Reply:
OK, but i really don't like this way.

Reply:
When i do in your way i raise
`hash key "title" is not a Symbol`

Reply:
`symbolize_keys` should help

Reply:
> `symbolize_keys` should help

This method exist only for ordinary `Hash`. I work with `ActionController::Parameters`. So, it raises `undefined method`.